share/lambda/Makefile

```diff
@@ -61,7 +61,7 @@ dist.zip: $(shell find src) | dist-pre dist dist-post
 		&& cd dist \
 		&& cp ../package*.json . \
 		&& npm install --production \
-		&& rm -rf package*.json node_modules/aws-sdk etc \
```

Comment:
What's the reason for removal? `aws-sdk` is already available in Lambda.

Reply:
To be honest, it was already on the original branch. I'll put it back and run the tests.

Reply:
Done.